.github/workflows/test-composite-actions.yaml

```
@@ -146,7 +146,8 @@ jobs:
       matrix:
         container:
           - ros:humble
-          - ghcr.io/autowarefoundation/autoware-universe:humble-latest
+          - ghcr.io/autowarefoundation/autoware:latest-autoware-universe
```

```suggestion
          - ghcr.io/autowarefoundation/autoware:latest-universe-devel
```

is better from today. 
https://github.com/autowarefoundation/autoware/pull/5170

---

Thank you @youtalk for your proposal. The fix will be applied in [this PR](https://github.com/autowarefoundation/autoware-github-actions/pull/319/) as this PR is merged.